index.js

```diff
@@ -1,14 +1,15 @@
 import express from 'express'
 
 import config from './config'
-import routeTest from './routes/test'
+import { roleRoute, testRoute } from './routes'

```

Comment:
Could you check if it will be possible to change testRoute into test and roleRoute into role?